concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetAccountInfoTest.java

```diff
@@ -0,0 +1,474 @@
+    private static final byte[] COMMITMENT_PRF = new byte[]{11, 11, 11};
+    private static final byte[] COMMITMENT_ID_CRED_SHARING_COEFF = new byte[]{12, 12, 12};
+    private static final byte[] COMMITMENT_ATTRIBUTE_0 = new byte[]{13, 13, 13};
+    private static final YearMonth POLICY_CREATED_AT
+            = YearMonth.newBuilder().setYear(2022).setMonth(10).build();
+    private static final YearMonth POLICY_VALID_TO
+            = YearMonth.newBuilder().setYear(2024).setMonth(10).build();
+    private static final byte[] FIRST_NAME_VALUE
+            = "policy-value-0".getBytes(StandardCharsets.UTF_8);
+    private static final byte[] ENC_ID_PUB_SHARE = new byte[]{90, 90, 90};
+    private static final NormalCredentialValues.Builder NORMAL_CREDENTIAL_BUILDER = NormalCredentialValues.newBuilder()
+            .setArThreshold(ArThreshold.newBuilder().setValue(AR_THRESHOLD).build())
+            .putAllArData(ImmutableMap.of(0, ChainArData.newBuilder()
+                    .setEncIdCredPubShare(ByteString.copyFrom(ENC_ID_PUB_SHARE))
+                    .build()))
+            .setCommitments(CredentialCommitments.newBuilder()
+                    .setCredCounter(toCommitment(COMMITMENT_CRED_COUNTER))
+                    .setMaxAccounts(toCommitment(COMMITMENT_MAX_ACCOUNTS))
+                    .setPrf(toCommitment(COMMITMENT_PRF))
+                    .addIdCredSecSharingCoeff(0, toCommitment(COMMITMENT_ID_CRED_SHARING_COEFF))
+                    .putAllAttributes(ImmutableMap.of(0, toCommitment(COMMITMENT_ATTRIBUTE_0)))
+                    .build())
+            .setCredId(CredentialRegistrationId.newBuilder()
+                    .setValue(ByteString.copyFrom(CREDENTIAL_REG_ID))
+                    .build())
+            .setIpId(IdentityProviderIdentity.newBuilder()
+                    .setValue(IDP_ID)
+                    .build())
+            .setKeys(CredentialPublicKeys.newBuilder()
+                    .setThreshold(SignatureThreshold.newBuilder().setValue(SIGNATURE_THRESHOLD).build())
+                    .putAllKeys(ImmutableMap.of(0, AccountVerifyKey.newBuilder()
+                            .setEd25519Key(ByteString.copyFrom(NORMAL_CRED_SIG_0))
+                            .build()))
+                    .build())
+            .setPolicy(Policy.newBuilder()
+                    .setCreatedAt(POLICY_CREATED_AT)
+                    .setValidTo(POLICY_VALID_TO)
+                    .putAllAttributes(ImmutableMap.of(
+                            FIRST_NAME.ordinal(),
+                            ByteString.copyFrom(FIRST_NAME_VALUE)))
+                    .build());
+
+    private static Commitment toCommitment(byte[] commitmentCredCounter) {
+        return Commitment.newBuilder()
+                .setValue(ByteString.copyFrom(commitmentCredCounter))
+                .build();
+    }
+
+    private static final Credential NORMAL_CREDENTIAL_CLIENT =
```

Comment:
```suggestion
    private static final Credential NORMAL_CREDENTIAL_EXPECTED =
```

Because this is the expectation right?

Reply:
removed and merge with the main Account Info Building / Instantiation. Since there was only a single usage